api_app/analyzers_manager/observable_analyzers/malprob.py

```diff
@@ -0,0 +1,137 @@
+import requests
+
+from api_app.analyzers_manager import classes
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+
+class MalprobSearch(classes.ObservableAnalyzer):
+    url: str = "https://malprob.io/api"
+    _api_key_name: str
+
+    def update(self):
+        pass
+
+    def run(self):
+        response = requests.get(
+            f"{self.url}/search/{self.observable_name}",
+            timeout=120,
```

Comment:
too high, I expect this to be fast

Reply:
should i completely remove timeout form here and directly put 10s in the soft t/o in migration file?

Reply:
you can do both cause if you don't set the timeout here, that would be none and the request could hang forever until the soft time limit is triggered. But triggering a soft time limit should be avoided if possible and the expected time required handled directly in the analyzer.